src/platforms/android/java/com/klippa/NativeScriptHTTP/Async.java

```diff
@@ -230,10 +209,56 @@ public static void ClearCookies() {
             }
         }
 
-        public static void AllowSslErrors(boolean allow) {
-            client = null;
-            allowSslErrors = allow;
+        public static void EnableSSLValidation() {
+            InitClient();
+            
+            try {
+                TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
```

Comment:
My intention was to do this inside `InitClient` while building the `client`, but before running `client.build()`